stratosphereprefs/Resources/Root.plist

```diff
@@ -21,6 +21,8 @@
 			<string>isEnabled</string>
 			<key>label</key>
 			<string>Enable</string>
+			<key>PostNotification</key>
```

Comment:
Have the settings app automatically shout at SpringBoard and let us know a setting changed.